CesiumGltfWriter/src/AccessorWriter.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include "CesiumGltf/Accessor.h"
+#include "CesiumGltf/AccessorSpec.h"
+#include "JsonWriter.h"
+
+namespace CesiumGltf {
+void writeAccessor(
+    const std::vector<Accessor>& accessors,
+    CesiumGltf::JsonWriter& jsonWriter);
+}
```

Comment:
Missing white line at the end of the file. Seems there is a bunch of them. Can you review all the missing files to add the newline? Thanks!